spec/domain/export/pdf/messages/letter_spec.rb

```diff
@@ -12,15 +12,48 @@
   let(:letter) { messages(:letter) }
   let(:recipients) { [people(:bottom_member)] }
   let(:options) { {} }
+  let(:layer) { groups(:top_layer) }
+  let(:group) { groups(:top_group) }
+  let(:list) { mailing_lists(:leaders) }
 
   subject { described_class.new(letter, recipients, options) }
+  # move list from top layer to top group
+  before { list.update!(group: group) }
```

Comment:
Könnte verwirrend sein wenn dieselbe Liste in dieser Spec an einer anderen Gruppe hängt als in anderen Specs... Warum nicht eine neue, semantisch benannte Liste in die Fixtures einfügen?

Reply:
oder eine Liste fabrizieren :wink: